programmers/K번째-수.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, commands) {
+    let answer = [];
+    let answerArray = [];
+    for(let i = 0; i < commands.length; i++) {
+        answerArray = array.slice(commands[i][0]-1, commands[i][1]).sort(function (a,b){return a-b;});
+        answer.push(answerArray[commands[i][2]-1])
+    }
```

Comment:
1. 문제에서 주어지는 i ,j, k 를 아래 처럼 선언하는것이 더욱 가독성이좋다.
2. sort 에서 펑션을 사용하였는데 arrow펑션을 사용하는것이 낫다.
3. return 이 한줄일때 생략가능
4.  공백 컨벤션을 지킬것.
```suggestion
   for (let idx = 0; idx < commands.length; idx++) {
        const [i, j, k] = [commands[idx][0] - 1, commands[idx][1], commands[idx][2] - 1];
        answers = array.slice(i, j).sort((a, b) => a - b);
        answer.push(answers[k])
    }
```